### ISSUE TITLE
Sphinx-intl

### ISSUE THREAD

**Issue by plaindocs** [sunpy/ablog#59]
Hi Folks,

Been using ablog over on writethedocs.org to great effect, thanks for that.

I'm looking at translation (https://github.com/writethedocs/www/issues/1155) and I'm not sure if this error is an ablog error or a sphinx-intl error.

`make gettext`

```
Exception occurred:
  File "/home/samuel/Repositories/WTD/www/venv/lib/python3.5/site-packages/ablog/post.py", line 355, in process_posts
    stdlabel = env.intersphinx_inventory.setdefault('std:label', {})
AttributeError: 'BuildEnvironment' object has no attribute 'intersphinx_inventory'
```

**Comment by nabobalis** [comment]
Hi @plaindocs, thanks for the report. I think we had a similar error occur on https://github.com/sunpy/ablog/issues/18 and was fixed (potentially) by adding "sphinx.ext.intersphinx" in the sphinx extensions list (in conf.py). I am not sure if this will fix your case, but it might be worth a try.

**Comment by plaindocs** [comment]
Thanks for the super speedy response!

Yeah, I should have noticed that I didn't have intersphinx included. Should probably be included in the sphinx-intl docs as a requirement.

**Comment by plaindocs** [comment]
It doesn't appear to be a sphinx-intl requirement, but an ablog requirement (https://github.com/sphinx-doc/sphinx/pull/7257#issuecomment-596033141). 

So I think we should add it here. Happy to make a PR for the docs if that would be well received.

**Comment by nabobalis** [comment]
Sure!

**Comment by nabobalis** [comment]
Closed by  #60
